homepage/migrations/0006_review_test_data.py

```diff
@@ -1,4 +1,5 @@
 from django.db import migrations, transaction
+NEW_IMAGE_FILE = 'images/new_test_image.jpg'
```

Comment:
please split it like
```suggestion
IMAGES_DIR = 'images'
IMAGE_FILE = 'test_image.jpg'
NEW_IMAGE_FILE = 'new_test_image.jpg'
```
now you can use it in your code below